### PR TITLE
Update /azure/fips copy

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

- Updated /azure/fips based on the [copy doc](https://docs.google.com/document/d/1Djxdzjy78bau8ELM4-MNGoM2YRa_QjpN1xJ4H4GjFyw/edit#)

## QA

- View the site in your web browser at: https://ubuntu-com-11444.demos.haus/azure/fips
- check the updates match the copy doc

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5008

### CODE REVIEW

File: templates/azure/fips.html [hunk 1]
```diff
@@ -123,12 +124,17 @@ <h2>Launch Ubuntu Pro FIPS for Azure</h2>
     </div>
   </div>
   <div class="row p-divider">
-    <div class="col-6 p-divider__block">
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--4">Ubuntu Pro FIPS 20.04 LTS</h3>
+      <p>Ubuntu Pro FIPS 20.04 LTS includes an azure-optimised FIPS-certified 5.4 kernel and other FIPS- certified modules pre-enabled out of the box. Ubuntu Pro FIPS provides extended security maintenance through April 2030.</p>
```
Comment (minkyngkm) [inline templates/azure/fips.html:129]:
```suggestion
      <p>Ubuntu Pro FIPS 20.04 LTS includes an azure-optimised FIPS-certified 5.4 kernel and other FIPS-certified modules pre-enabled out of the box. Ubuntu Pro FIPS provides extended security maintenance through April 2030.</p>
```

File: templates/azure/fips.html [hunk 2]
```diff
@@ -123,12 +124,17 @@ <h2>Launch Ubuntu Pro FIPS for Azure</h2>
...
+    </div>
+    <div class="col-4 p-divider__block">
       <h3 class="p-heading--4">Ubuntu Pro FIPS 18.04 LTS</h3>
       <p>Ubuntu Pro FIPS 18.04 LTS includes an Azure-optimised FIPS-certified 4.15 kernel and other FIPS certified modules pre-enabled out of the box. Ubuntu Pro FIPS provides extended security maintenance through April 2028.</p>
       <p><a href="https://azuremarketplace.microsoft.com/en-ca/marketplace/apps/canonical.0001-com-ubuntu-pro-bionic-fips" class="p-button--positive"><span>Launch now</span></a></p>
     </div>
-    <div class="col-6 p-divider__block">
+    <div class="col-4 p-divider__block">
       <h3 class="p-heading--4">Ubuntu Pro FIPS 16.04 LTS</h3>
       <p>Ubuntu 16.04 LTS includes FIPS-certified 4.4  kernel and other FIPS certified modules pre-enabled out of the box. Ubuntu Pro FIPS provides extended security maintenance through April 2026.</p>
```
Comment (minkyngkm) [inline templates/azure/fips.html:139]:
```suggestion
      <p>Ubuntu 16.04 LTS includes FIPS-certified 4.4  kernel and other FIPS-certified modules pre-enabled out of the box. Ubuntu Pro FIPS provides extended security maintenance through April 2026.</p>
```

File: templates/azure/fips.html [hunk 3]
```diff
@@ -347,7 +395,7 @@ <h2>Pricing</h2>
           </tr>
         </tbody>
       </table>
-      <p>For complete pricing, see <a href="https://azuremarketplace.microsoft.com/en-ca/marketplace/apps/canonical.0001-com-ubuntu-pro-bionic-fips">Ubuntu Pro FIPS 18.04 LTS on the Azure Marketplace</a></p>
+      <p>For complete pricing, see <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-pro-focal-fips">Ubuntu Pro FIPS 20.04 LTS on the Azure Marketplace</a></p>
```
Comment (minkyngkm) [inline templates/azure/fips.html:398]: `>` is missing here and in line 318
```suggestion
      <p>For complete pricing, see <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/canonical.0001-com-ubuntu-pro-focal-fips">Ubuntu Pro FIPS 20.04 LTS on the Azure Marketplace&nbsp;&rsaquo;</a></p>
```

File: templates/azure/fips.html [hunk 4]
```diff
@@ -414,7 +465,7 @@ <h2>Getting the best of Ubuntu Pro for Azure</h2>
         }}
       </a>
       <h4>Operating open source securely</h4>
-      <p>It's no longer just Linux, Apache, MySQL and PHP — open source ecosystems like Java, NodeJS, Python and Go are everywhere in the enterprise, as are modern workloads like Kafka, Elasticsearch and Cassandra. In opting for open source, development teams in every organisation are optimising for velocity, flexibility and ideal economics, and the result has been an incredible ramp up in software product delivery.</p>
+      <p>It's no longer just Linux, Apache, MySQL and PHP — open source ecosystems like Java, NodeJS, Python and Go are everywhere in the enterprise, as are modern workloads like Kafka, Elasticsearch and Cassandra. In opting for open source, development teams in every organisation are optimising for velocity, flexibility and ideal economics, and the result has been an incredible ramp-up in software product delivery.</p>
```
Comment (minkyngkm) [inline templates/azure/fips.html:468]:
```suggestion
      <p>It's no longer just Linux, Apache, MySQL and PHP &mdash; open source ecosystems like Java, NodeJS, Python and Go are everywhere in the enterprise, as are modern workloads like Kafka, Elasticsearch and Cassandra. In opting for open source, development teams in every organisation are optimising for velocity, flexibility and ideal economics, and the result has been an incredible ramp-up in software product delivery.</p>
```

**Comment by sowasred2012** [comment]
@minkyngkm ah nice catch - updated.

**Pull request merged**